govuk-social-providers-only/login/login.ftl

```diff
@@ -75,5 +69,12 @@
                 </ul>
             </div>
         </#if>
+
+        <#if realm.password && realm.registrationAllowed && !usernameEditDisabled??>
+            <div id="kc-registration">
+                <h2 class="heading-medium">${msg("noAccount")}</h2>
+                <span><a href="${url.registrationUrl}">${msg("registerLink")}</a></span>
+            </div>
+        </#if>
```

Comment:
Oh yes I see, sorry — quite right, that makes more sense.

Yeah I think the span is just there because I copied the original Keycloak base theme, and there it might have been used by the CSS.